toolkit/scripts/setuplkgtoolchain.sh

```diff
@@ -41,6 +45,8 @@ check_for_modified_manifests
 
 update_manifests
 
+delete_lkg
```

Comment:
Could we use `trap` instead for all clean-up steps? This way we make sure to always perform the clean-up regardless if the script finishes or not. It can be interrupted by an error from an earlier command from by user input. Take a look at `toolkit/scripts/build_cargo_cache.sh` for an example how to use `trap`.

Reply:
Thanks, great suggestion